D3Charts/AncestralFanChart.py

```diff
@@ -398,7 +398,7 @@ def write_report(self):
                 fp.write(outstr)
 
         except IOError as msg:
-            ErrorDialog(_("Failed writing %s: %s") % (self.desthtml, str(msg)),
+            ErrorDialog(_("Failure writing {target_path}: {message}").format(target_path=self.desthtml, messgae=str(msg)),
```

Comment:
@Nick-Hall Typo "messgae" instead of "message"? Also in line 648.

Reply:
Yes, it looks like a typo.

Reply:
Added PR to fix Ancestral Fan Chart and Descendant Indented Tree typo errors. Published PR.